StereoKitC/asset_types/texture_.h

```diff
@@ -6,7 +6,7 @@ namespace sk {
 
 void        tex_set_zbuffer      (tex_t texture, tex_t depth_texture);
 void        tex_set_options      (tex_t texture, tex_sample_ sample = tex_sample_linear, tex_address_ address_mode = tex_address_wrap, int32_t anisotropy_level = 4);
-void        tex_set_surface      (tex_t texture, void *native_surface, tex_type_ type, int64_t native_fmt, int32_t width, int32_t height, int32_t surface_count);
+void        tex_set_surface      (tex_t texture, void *native_surface, bool32_t owned, tex_type_ type, int64_t native_fmt, int32_t width, int32_t height, int32_t surface_count);
```

Comment:
This function prototype is actually a duplicate of the one in stereokit.h! This should probably be removed entirely :)

Reply:
i figured, why is that header file there and can i just yeet the whole header?

Reply:
The header file itself is important, there's some items separated out here into different groups to help with improving build time :)